uk.co.compucorp.civicrm.tasksassignments/views/dashboard/documents.html

```diff
@@ -134,7 +136,8 @@ <h3 class="panel-title">&nbsp;</h3>
         </div>
       </div>
       <div class="panel-body">
-        <div class="{{prefix}}documents-container" ct-spinner ct-spinner-id="list">
+        <div class="text-center" ng-if="listPaginated.length === 0">There are no documents</div>
```

Comment:
can't you use a ng-switch in the parent?

Reply:
I have changed the structure of the HTML, now the ng-if and the ng-show are not under same parent, so cant use ng-switch. The structure was changed because `{{prefix}}list-no-result` has a margin top, and `panel-body` also has padding top, so keeping `{{prefix}}list-no-result` inside creates double spacing.